core/domain/exp_jobs_one_off_test.py

```diff
@@ -526,7 +526,7 @@ def test_reverts_not_counted(self):
 
         user_a_id = self.get_user_id_from_email(self.EMAIL_A)
 
-        # Let USER A make 2 non-revert commits and 1 revert
+        # Let USER A make 3 non-revert commits and 1 revert
```

Comment:
Ideally it should be dropped because the "revert" code has a comment of its own (`# Let USER A revert version 3 to version 2`). However, it can be argued that both the parts of code (non-revert commits and revert) are subsumed under `# Let USER A make 3 non-revert commits and 1 revert`.